### ISSUE TITLE
Can act works with Containers in matrix ?

### ISSUE THREAD

**Issue by kozicpetar** [nektos/act#376]
Hi, I can't find anything does act support container images and can I use it in matrix?
On example, if I try this:

```
jobs:
  test:
    name: Test
    strategy:
      matrix:
        os: ['ubuntu:18.04', 'debian:10']
    runs-on: ubuntu-latest
    container: ${{ matrix.os }}
```

I get error:
`Error: invalid reference format`

OS: `MacOS`
Act version: `act version 0.2.14`

**Comment by SteffenSeckler** [comment]
I am getting the same error.

act version: 0.2.17

**Comment by SteffenSeckler** [comment]
Should be fixes in #413.
